ipynb/android-wloads/Android_PCMark_Tests.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# EAS Testing - PCMark benchmark on Android"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this experiment is to run benchmarks on a Nexus N5X running Android with an EAS kernel and collect results. The analysis phase will consist in comparing EAS with other schedulers, that is comparing *sched* governor with:\n",
+    "\n",
+    "    - interactive\n",
+    "    - performance\n",
+    "    - powersave\n",
+    "    - ondemand\n",
+    "    \n",
+    "The benchmark we will be using is ***PCMark*** (https://www.futuremark.com/benchmarks/pcmark-android). You will need to **manually install** the app on the Android device in order to run this Notebook.\n",
+    "\n",
+    "When opinening PCMark for the first time you will need to Install the work benchmark from inside the app."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "reload(logging)\n",
+    "log_fmt = '%(asctime)-9s %(levelname)-8s: %(message)s'\n",
+    "logging.basicConfig(format=log_fmt)\n",
+    "\n",
+    "# Change to info once the notebook runs ok\n",
+    "logging.getLogger().setLevel(logging.INFO)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Populating the interactive namespace from numpy and matplotlib\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pylab inline\n",
+    "\n",
+    "import copy\n",
+    "import os\n",
+    "from time import sleep\n",
+    "from subprocess import Popen\n",
+    "\n",
+    "# Support to access the remote target\n",
+    "import devlib\n",
+    "from env import TestEnv\n",
+    "\n",
+    "# Support for trace events analysis\n",
+    "from trace import Trace\n",
+    "from trace_analysis import TraceAnalysis\n",
+    "\n",
+    "# Suport for FTrace events parsing and visualization\n",
+    "import trappy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test Environment set up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In case more than one Android device are conencted to the host, you must specify the ID of the device you want to target in `my_target_conf`. Run `adb devices` on your host to get the ID."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Setup a target configuration\n",
+    "my_target_conf = {\n",
+    "    \n",
+    "    # Target platform and board\n",
+    "    \"platform\"    : 'android',\n",
+    "    \n",
+    "    # Device ID\n",
+    "    \"device\" : \"00b1346f0878ccb1\",\n",
```

Comment:
Would suggest to keep this device option commented.
In the common case we have just one device connected, this this allows to run the notebook on the default device without needs to update the notebook.